.github/workflows/release.yml

```diff
@@ -11,7 +11,7 @@ on:
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 env:
-  RUST_VERSION: 1.71
+  RUST_VERSION: 1.73
```

Comment:
What is the reason for the bump here?  It's drifting from the [current MSRV](https://github.com/fermyon/spin/blob/main/Cargo.toml#L15).